.env.docker.e2e.example

```
@@ -15,6 +15,7 @@ DB_DECIMAL_NUMBERS=true
 # Directus
 # https://docs.directus.io/self-hosted/config-options.html#general
 PUBLIC_URL=http://localhost:28055
+DASH_URL=http://localhost:13010
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DASH_URL key should go before the PUBLIC_URL key

(UnorderedKey)